docs/src/3.16FlowTypes.mdx

```diff
@@ -0,0 +1,38 @@
+import GithubLink from './utils/GithubLink'
+
+# Flow types
+
+Flow types are supported. To use the type, import directly from `regl-worldview`. More type definitions: <GithubLink href="https://github.com/cruise-automation/webviz/blob/master/packages/regl-worldview/src/types/index.js" />
```

Comment:
"To use the type, import directly from" => "To use types, import them directly from"?